### ISSUE TITLE
Info file not being included in Info list automatically

### ISSUE THREAD

**Issue by verdammelt** [pprevos/denote-explore#27]
It looks like this package has an Info file but I was not seeing it in my Info's top directory node. It appears that it was not because the info file is missing a block at the top such as:

```
INFO-DIR-SECTION Emacs
START-INFO-DIR-ENTRY
* Denote Explore: (denote-explore).       Explore your Denote digital garden
END-INFO-DIR-ENTRY
```

(I experimented with this after noticing that other packages I installed had their info files automatically added to the top node.)

I am not 100% positive but it would seem that to get this block in an info file adding a block such as the following into the texi file may do the trick (again by looking at other libraries this seems to be the way they work...)

```
@dircategory Emacs
@direntry
* Denote Explore: (denote-explore).     Explore your Denote digital garden
@end direntry
```

I'd be happy to create a PR with this change but I am unsure how to test that it would work properly.

**Comment by pprevos** [comment]
I wrote the manual in Org mode and export it to Info mode. Is there any good documentation on best practice for this?

Happy to receive a PR for the Org file because the info file will be overwritten.

**Comment by verdammelt** [comment]
> I wrote the manual in Org mode and export it to Info mode. Is there any good documentation on best practice for this?
> 
Hmm good question, I have no idea :( I'll see if I can find out any info.

**Comment by verdammelt** [comment]
I found the documentation: org-link: `info:org#Info directory file` which is section `13.14.5 Info directory file` in my org-mode info file.

It says to include directives like:

```
     #+TEXINFO_DIR_CATEGORY: Emacs
     #+TEXINFO_DIR_NAME: Org Mode
     #+TEXINFO_DIR_DESC: Outline-based notes management and organizer

     #+TEXINFO_DIR_CATEGORY: Emacs
     #+TEXINFO_DIR_NAME: Org Mode
     #+TEXINFO_DIR_DESC: Outline-based notes management and organizer
```

I'll see about creating a PR to make this change.

**Comment by pprevos** [comment]
Thanks for the update. Resolved with [1e35592](https://github.com/pprevos/denote-explore/commit/1e35592a55d6c8ba2eaf63862544a2b86f187a58)
 ·
